app/labor/email_digest.rb

```diff
@@ -8,19 +8,8 @@ def initialize(users = [])
   end
 
   def send_periodic_digest_email
-    @users.find_each do |user|
-      user_email_heuristic = EmailLogic.new(user).analyze
-      next unless user_email_heuristic.should_receive_email?
-
-      articles = user_email_heuristic.articles_to_send
-      begin
-        next unless user.email_digest_periodic?
-
-        DigestMailer.with(user: user, articles: articles).digest_email.deliver_now
-      rescue StandardError => e
-        Honeybadger.context({ user_id: user.id, article_ids: articles.map(&:id) })
-        Honeybadger.notify(e)
-      end
+    @users.ids.each do |user_id|
```

Comment:
Question: do you think there's a benefit in loading ids in batches? Something like `User.select(:id).in_batches` ?

It's maybe a premature optimization though